docs/tutorials/gluon/fit_api_tutorial.md

```diff
@@ -156,6 +156,31 @@ Fit API is also customizable with several `Event Handlers` which give a fine gra
 
 One can use built-in event handlers such as `LoggingHandler`, `CheckpointHandler` or `EarlyStoppingHandler` to log and save the model at certain timesteps during training and stopping the training when the model's performance plateaus. One can also create a custom handler by inheriting [`EventHandler`](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/estimator/event_handler.py#L31).
 
+### Custom Event Handler
+
+Here we will showcase an example to create a custom event handler by inheriting from `EventHandler` class. Our custom event handler is a simple one, that just records the loss values at the end of every epoch in our training phase.
```

Comment:
can you add a Note below: 

.Note... The EventHandler holds a  reference to the `Estimator` object. The Estimator object reference in the handler is updated when Estimator.Fit API is called.

Reply:
Done.